### PR TITLE
BPI-776: changed confusing references of {core_id} and {tenant_id} to…

### PR DESCRIPTION
**Pull request by scotchneat**
… {device_id} and {account_id} respectively since this is not a traditional cloud api.

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-fd9b7bb8a7/docs/dedicated-load-balancers/v2/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-af2cda767d/docs/dedicated-load-balancers/v2/)

File: load-balancer/api-docs/common-gs/auth-using-curl.rst [hunk 1]
```diff
@@ -137,7 +137,7 @@ request, prefix the variable name with a ``$``, for example ``$ENDPOINT``.
 
    .. code::
 
-      $ export TENANT_ID="tenant-id"
+      $ export ACCOUNT_ID="account-id"
```
Comment (rajeevkct) [inline load-balancer/api-docs/common-gs/auth-using-curl.rst]: Line 67 says  tenant_id is the account number, i think we  can keep it as tenant_id here.

**Review by rajeevkct — CHANGES_REQUESTED** [review]

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-258a0e82f7/docs/dedicated-load-balancers/v2/)

File: load-balancer/api-docs/common-gs/auth-using-curl.rst (hunk 1 above)
Comment (scotchneat) [reply load-balancer/api-docs/common-gs/auth-using-curl.rst]: addressed these concerns and reverted the name to tenant

**Review by cyrichardson — APPROVED** [review]
From a copy edit perspective, LGTM. Thanks! Probably need a subject matter expert to do a technical review. I see you have requested one. Yay!

**Review by rajeevkct — APPROVED** [review]

**Comment by ramanbhati** [comment]
Merging.

**Pull request merged**